app/components/primer/alpha/action_list.rb

```diff
@@ -73,6 +73,7 @@ def initialize(
           system_arguments[:classes],
           "ActionListWrap",
           "ActionListWrap--subGroup",
+          "ActionListItem--hasSubItem",
```

Comment:
I'm confused about the addition of classes here. This case should only have `ActionListWrap` for example 
![image](https://user-images.githubusercontent.com/18661030/220996704-b12e6260-6968-4683-b94e-fa4ae8c3da1f.png)

`subGroup` and `hasSubItem` are all used for collapsible sections

Reply:
I think we added these here because they didn't seem to adversely affect the rendered result and because they were difficult to add elsewhere. Let me take another stab at this.